home/templates/home/login.html

```diff
@@ -3,10 +3,12 @@
 {% block title %}Login{% endblock %}
 
 {% block content %}
+<body class="main-bg">
```

Comment:
@idokk1 
I recommend that you also add margins so that this screen will look better . 
You can do this by adding for example:
`class ="m-4" `

Reply:
added